packages/perseus-editor/src/widgets/expression-editor.tsx

```diff
@@ -18,7 +18,7 @@ const {InfoTip, PropCheckBox, TexButtons} = components;
 const {getDependencies} = Dependencies;
 
 // An answer can be considered correct, wrong, or ungraded.
-const CONSIDERED = ["correct", "wrong", "ungraded"];
+const CONSIDERED = ["correct", "wrong", "ungraded"] as const;
```

Comment:
Can we use this pattern to re-use this type in `perseus-types.ts`? Or the other way around-- define there and use here?

Names are just to demonstrate.
```
export const consideredStrings = ["correct", "wrong", "ungraded"] as const;
export type considered = typeof consideredStrings[number];
```